docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,243 @@
 ---
-title: Getting Started with Desktop Access
+title: Get started with Desktop Access
 description: Passwordless access to Windows systems for local users
 videoBanner: 9DyKQbg4ORc
 ---
 
-This guide will help you configure Teleport to provide secure, passwordless access
-to Windows desktops. This configuration does not require an Active Directory domain.
+This guide demonstrates how to configure Teleport to provide secure, passwordless access
+to Microsoft Windows desktops that aren't joined to an Active Directory domain.
+Passwordless access is only supported for **commercial editions** of Teleport.
+For the open source Teleport Community Edition, you can protect Microsoft Windows servers 
+and desktops by integrating Teleport with Active Directory. 
 
-<Admonition type="note">
-Passwordless access for local users is an Enterprise-only feature.
+Integrating with Active Directory enables you to automatically discover Microsoft Windows servers 
+and desktops that are joined to Active Directory domains. If you use a commercial edition of 
+Teleport can also mix the automatic discovery provided by Active Directory integration
+with the static host definitions demonstrated in this guide.
 
-For open source Teleport, consider integrating Teleport with Active Directory
-for automatic discovery by reading [Desktop Access with Active Directory](./active-directory.mdx).
-
-Teleport Enterprise users can also mix the Teleport Active Directory integration
-with the static host definitions described below.
-</Admonition>
-
-<Details
-  title="Version warning"
-  opened={true}
-  min="12.0"
->
-Passwordless access for local users is available starting from Teleport `v12`.
-Previous versions of Teleport can implement Windows Access by integrating with
-an [Active Directory domain](./active-directory.mdx).
-</Details>
+For information about integrating Teleport with Active Directory, see [Desktop Access with 
+Active Directory](./active-directory.mdx).
 
 ## Prerequisites
 
+To complete the steps in this guide, verify your environment meets the following requirements:
+
 (!docs/pages/includes/commercial-prereqs-tabs.mdx!)
 
-- A Linux server to run the Teleport Desktop Service on.
+- A Linux server to run the Teleport Windows Desktop Service.
   You can reuse an existing server running any other Teleport instance.
-- A server or virtual machine running a Windows operating system with
-  Remote Desktop enabled and the RDP port available to the Linux server.
+- A physical or virtual machine running Microsoft Windows with
+  Remote Desktop enabled and the RDP port—typically, port 3389—open for inbound connections
+  from the Linux server.
 - (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/4. Prepare Windows
+## Step 1/4. Prepare Microsoft Windows desktop
```

Comment:
It's probably not necessary, but it's no uncommon. I think mainly because windows is a common word that required "Microsoft" to be branded whereas macOS can be brand identified without Apple. I'm okay with removing the Microsoft part but that's why I included it.